testing/test_conftest.py

```diff
@@ -90,19 +91,19 @@ def test_conftest_in_nonpkg_with_init(tmpdir, _sys_snapshot):
     ConftestWithSetinitial(tmpdir.join("adir-1.0", "b"))
 
 
-def test_doubledash_considered(testdir):
-    conf = testdir.mkdir("--option")
+def test_doubledash_considered(pytester: Pytester) -> None:
+    conf = pytester.mkdir("--option")
     conf.ensure("conftest.py")
```

Comment:
```Path``` dosen't have ```ensure``` attribute. If you want to ensure a single file then you can acheive it by using the following method. 

```suggestion
    conf.joinpath("conftest.py").touch()
```